sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -56,6 +56,7 @@ class ParquetFileFormat
   with DataSourceRegister
   with Logging
   with Serializable {
+  ParquetFileFormat.redirectParquetLogs
```

Comment:
You should just be able to put this in the class init for `ParquetFileFormat`, by adding this to the body of `object ParquetFileFormat` below.